content/02.main-text.md

```diff
@@ -2,34 +2,31 @@
+In January 2018, the article was accepted by the journal that it was initially submitted to.
+We'll discuss our experience with the Deep Review project including the pros and cons of open collaborative reviews, as well as the infrastructure we created, termed Manubot, to enable open manuscript writing online.
+We also discuss the extent to which such efforts can contribute to a living and frequently updated literature. `todo: discuss stage 2 maintainers?`
 
-## The Deep Review
-
-We initiated the Deep Review by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
+We initiated our open online review article by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
 GitHub is a platform designed for collaborative software development that's adaptable for collaborative writing.
 From the start, we made the GitHub repository public, applying a Creative Commons Attribution [License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md) to the manuscript.
 Next, we encouraged anyone interested to contribute by proposing changes or additions.
 Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media, deciding without solicitation to contribute.
-In total, the Deep Review attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
-
-To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot) tool for automating manuscript generation.
-
-## Results
+In total, the project that we termed the "Deep Review" attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
 
 Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
-
 `TODO: need archival issue link`
 In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing the reviews with potential manuscript authorship.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
+To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot) tool for automating manuscript generation.
```

Comment:
https://github.com/greenelab/manubot-rootstock is currently a better homepage than https://github.com/greenelab/manubut